README.md

```diff
@@ -1,27 +1,119 @@
 # Common Workflow Language Website 2.0
 
-A Rewrite and expansion of the [CWL Website](https://commonwl.org), using Jekyll. The current staging site is https://github.com/lunacodes/cwl-staging
+<!-- MarkdownTOC -->
 
-## Staging Issues
+* [Vision for the CWL Project](#vision-for-the-cwl-project)
+    * [Mission of the CWL Project](#mission-of-the-cwl-project)
+* [Editing the Site - General Flow](#editing-the-site---general-flow)
+    * [Config Files & Local Development](#config-files--local-development)
+* [Directory Structure](#directory-structure)
+* [Video Player - Plyr](#video-player---plyr)
+    * [Plyr API options](#plyr-api-options)
+* [Twitter Feed](#twitter-feed)
+* [SEO Tags](#seo-tags)
 
-* Top Nav needs to re-integrate logic checks for http and https
+<!-- /MarkdownTOC -->
 
-## Video Player
+This is the repo for the [CWL Website](https://commonwl.org).
 
-* plyr.js
-* Homepage video player code in `_includes/home/video-player.html`
-* Move progress bar above controls?
-* Change background color on Play, to avoid whitewash
+* GitHub Repo: https://github.com/common-workflow-language/
+* Staging Branch: https://github.com/common-workflow-language/cwl-website/tree/add-jekyll-site
+* Staging Site: https://www.commonwl.org/cwl-staging/
 
-### Subtitles
+<a id="vision-for-the-cwl-project"></a>
+## Vision for the CWL Project
+_What this community hopes to have accomplished in the future_
```

Comment:
Yay! Though the mission and vision statements shouldn't be part of this PR, and they need to be part of the content of www.commonwl.org and not the README for this repo

Reply:
Ahhhh whoops - that makes more sense! I created a PR (https://github.com/common-workflow-language/cwl-website/pull/62) for it, and will delete from here.

Reply:
That PR was merged, so please delete the mission and vision statement from here.